bin/release

```diff
@@ -19,6 +19,8 @@ rake release
 
 docker build --rm -t codeclimate/codeclimate .
 docker push codeclimate/codeclimate
+docker tag codeclimate/codeclimate "codeclimate/codeclimate:$version"
+docker push "codeclimate/codeclimate:$version"
```

Comment:
Agreeing with @GordonDiggs on this: I think it's better to be explicit and only push the release we care about.